migrations/20180725202700-populate-collective-with-opencollective-bot.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const Promise = require('bluebird');
+
+const insert = (sequelize, table, entry) => {
+  console.log(sequelize);
+  return sequelize.query(`
+    INSERT INTO "${table}" ("${Object.keys(entry).join('","')}") VALUES (:${Object.keys(entry).join(",:")})
+  `, { replacements: entry });
```

Comment:
Can we figure a better syntax, using sequelize insert?

Reply:
I think we can, tbh I kinda copied the way the insertions were done from past [migration files](https://github.com/opencollective/opencollective-api/tree/master/migrations)(like [here](https://github.com/opencollective/opencollective-api/blob/master/migrations/201709110000-DoubleEntry.js#L18-L27) and [here](https://github.com/opencollective/opencollective-api/blob/master/migrations/201707140000-GroupToCollective.js#L99-L103) for example) as most of them use the same function(which actually could be an util function or something like that as the same code is there for several times). But yeah I will definitely find an easier way to do it. thanks

Reply:
that's the helper function that I wrote in the past. It would be indeed better to use the `sequelize.insert`. I'm not sure anymore why I did it this way. It was a long time ago and then I kept on copy pasting what worked. I think it might have to do with the ability to do a `DRY_RUN` and stack all queries before running them and/or with more control on the actual data being recorded in the database. Can't remember for sure. But since migration files are often about raw queries, this helper function has been quite helpful (and might be worth pulling it into a migration lib)